exporter/exporter.go

```diff
@@ -127,8 +127,7 @@ func (e *Exporter) Run() {
 		ErrorLog:      e.logger,
 	})
 
-	addr := fmt.Sprintf(":%d", e.port)
-	exporter_shared.RunServer("MongoDB", addr, e.path, handler)
+	exporter_shared.RunServer("MongoDB", e.webListenAddress, e.path, handler)
```

Comment:
Sorry for asking that question again, but how can that work, given that exporter_shared depends on kingpin? @BupycHuk

Reply:
Because we are using environment variables to set HTTP auth, not command line parameters. 
The command line parameters we use are being set via Kong in e.weblistenaddress and path.

Reply:
Tested AUTH by setting `export HTTP_AUTH=zapp:brannigan`
then ran the exporter and it asked me for user/pass to get the metrics and `zapp`:`brannigan` worked

Reply:
Thank you